### PR TITLE
CHANGELOG updates for 1.7.2

### PR DESCRIPTION
**Pull request by andreeis**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,15 @@
 # What's New?
 
+## 1.7.2
+Bug fixes:
+- Fix paths of target sources outside the workspace. [#1504](https://github.com/microsoft/vscode-cmake-tools/issues/1504) [@sleiner](https://github.com/sleiner)
+- Use stricter type checks in presets expansion. [#1815](https://github.com/microsoft/vscode-cmake-tools/issues/1815)
+- Solve conflict between -DCMAKE_GENERAOR:STRING=Ninja versus -G "Visual Studio 16 2019" -A x64. [PR #1753](https://github.com/microsoft/vscode-cmake-tools/pull/1753) [@lygstate](https://github.com/lygstate)
+- Fix operator precedence when getting code page. [#1615](https://github.com/microsoft/vscode-cmake-tools/issues/1615) [@taoyouh](https://github.com/taoyouh)
+- Override the locale when querying compiler versions. [#1821](https://github.com/microsoft/vscode-cmake-tools/issues/1821)
+- Fix typo in CMakePresets.json schema. [PR #1809](https://github.com/microsoft/vscode-cmake-tools/pull/1809) [@1809]
```
Comment (bobbrow) [inline CHANGELOG.md:10]: This one is missing the contributor alias.

Comment (bobbrow) [reply CHANGELOG.md:10]: bluec0re

Comment (andreeis) [reply CHANGELOG.md:10]: Great catch, thx.

Comment (bobbrow) [reply CHANGELOG.md]: There is still a stray `[@1809]` in there.  Remove it and then this is good to merge.

**Review by bobbrow — APPROVED** [review]

**Pull request merged**